sys/readFileSyncBrowser.js

```diff
@@ -1,5 +1,5 @@
 const { getFile } = require('./files');
 
-const readFileSync = (path, options) => getFile(path).data;
+const readFileSync = (path, options) => getFile(path);
 
 module.exports.readFileSync = readFileSync;
```

Comment:
The readFileSync change will break on node.

Otherwise looks good.

Reply:
So if I understand correctly right now we're in a spot where as long as a path is written to from the browser and then read from the browser everything is OK. If a path is written to from node and then read from node everything is OK. But if we were to somehow cross them then things would break, right? In theory that's not something which will happen, but it still seems wrong to have a function with the same name do different things in the browser and in node.

What do you think the solution is? Should I modify the node code to write an object with a translator to the fs? That doesn't seem like what we want...